samples/CarterSample/Startup.cs

```diff
@@ -17,7 +13,11 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddSingleton<IActorProvider, ActorProvider>();
 
-            services.AddCarter();
+            services.AddCarter(options => options.OpenApi = new OpenApiOptions("Carter <3 OpenApi", new[] { "http://localhost:5000" }, new Dictionary<string, OpenApiSecurity>
```

Comment:
Don't make this an expression. It looks noisy and the line is too long.

Reply:
Pass an instance in instead?

Reply:
I think so.

Reply:
Wait, no, I meant the OpenAPI part.

Reply:
done on a new commit!